### PR TITLE
create home page

### PR DESCRIPTION
**Pull request by Roman240594**

### CODE REVIEW

File: app/scss/modules/_contact.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        align-items: center;
+        color: rgba(51, 51, 51, 0.5);
+        display: inline-block;
+        padding-top: 25px;
+        padding-left: 40px;
+        transition: all .5s;
+        &:hover {
+            color: $color-corn;
+        }
+        // &-gps {
```
Comment (jules-js) [inline app/scss/modules/_contact.scss]: commented code - for what?

File: app/index.html [hunk 2]
```diff
@@ -5,12 +5,489 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="assets/css/style.css">
-    <title>Document</title>
+    <link rel="icon" href="favicon.png">
+    <link rel="stylesheet" href="css/slick.css">
+    <link rel="stylesheet" href="css/main.css">
+    <title>HUNGRYPEO</title>
```
Comment (jules-js) [inline app/index.html]: maybe "HUNGRY PEOPLE"?

File: app/index.html [hunk 3]
```diff
@@ -5,12 +5,489 @@
...
 </head>
 
 <body>
+    <header class="header">
```
Comment (jules-js) [inline app/index.html:15]: why header contants banner too?

File: app/index.html [hunk 4]
```diff
@@ -5,12 +5,489 @@
...
+                        <li class="header__nav-item"><a class="header__nav-link uppercase text-center"
+                                href="#events">EVENTS</a></li>
+                        <li class="header__nav-item"><a class="header__nav-link uppercase text-center"
+                                href="#contact">CONTACT</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="header__banner flex flex_space-between flex_align-center">
+                <div class="schedule header__schedule">
+                    Mon - Fri: 8PM - 10PM, Sat - Sun: 8PM - 3AM
```
Comment (jules-js) [inline app/index.html]: text-orientation CSS property

File: app/index.html [hunk 5]
```diff
@@ -5,12 +5,489 @@
...
+                </div>
+                <div class="header__content flex flex_column text-center flex_self-start">
+                    <h6 class="header__sup-title uppercase">
```
Comment (jules-js) [inline app/index.html]: subtitle :)

File: app/index.html [hunk 6]
```diff
@@ -5,12 +5,489 @@
...
+                        hungry people
+                    </h1>
+                    <div class="header__buttons">
+                        <a class="button button_transparent uppercase header__button" href="#book">BOOK TABLE</a>
+                        <a class="button button_transparent uppercase header__button" href="#">EXPLORE</a>
+                    </div>
 
+                </div>
+                <div class="header__contacts flex flex_column">
+                    <a href="#"><img class="header__contact-logo" src="assets/images/facebook.png" alt="facebook"></a>
```
Comment (jules-js) [inline app/index.html]: check clickable area of link, it's wrong

File: app/scss/modules/_about.scss [hunk 7]
```diff
@@ -0,0 +1,13 @@
+.about {
```
Comment (jules-js) [inline app/scss/modules/_about.scss:1]: why don't you add just padding top and bottom to this section?

File: app/scss/modules/_header.scss [hunk 8]
```diff
@@ -0,0 +1,94 @@
+.header {
+    background-image: url(../assets/images/header-bg.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+    color: $color-white;
+    margin-bottom: 100px;
```
Comment (jules-js) [inline app/scss/modules/_header.scss]: remove it please

File: app/scss/modules/_about.scss [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    margin-bottom: 100px;
+    &__content{
+        padding-top: 117px;
+        max-width: 600px;
+    }
+    &__title::after {
+        left: 270px;
```
Comment (jules-js) [inline app/scss/modules/_about.scss]: bad idea (hardcode) to center line

Comment (jules-js) [reply app/scss/modules/_about.scss]:
```
left: 0;
right: 0;
margin: 0 auto
```

File: app/scss/base/_base.scss [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+    }
+    &_left {
+        left: 0;
+        top: 105px;
+    }
+}
+
+.image {
+    max-width: 475px;
+    background: transparent;
```
Comment (jules-js) [inline app/scss/base/_base.scss]: for what?

File: app/index.html [hunk 11]
```diff
@@ -5,12 +5,489 @@
...
+                            a, tempor magna.
+                        </div>
+                        <div class="text text__light about__text">
+                            Integer ullamcorper neque eu purus euismod, ac faucibus mauris posuere. Morbi non ultrices
+                            ligula. Sed dictum, enim sed ullamcorper feugiat, dui odio vehicula eros, a sollicitudin
+                            lorem quam nec sem. Mauris tincidunt feugiat diam convallis pharetra. Nulla facilisis semper
+                            laoreet.
+                        </div>
+                    </div>
+                    <div class="about__img flex_basis-50">
```
Comment (jules-js) [inline app/index.html]: центрувати відносно блоку, який відповідає за колонку так собі ідея
я б огорнула картинку у div, а квадратик зробила псевдоелементом з абсолютним позиціонуванням

File: app/scss/modules/_team.scss [hunk 12]
```diff
@@ -0,0 +1,16 @@
+.team {
+    background-image: url(../assets/images/team-bg.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
```
Comment (jules-js) [inline app/scss/modules/_team.scss]: not according to the design, for what 100vh?

File: app/scss/modules/_team.scss [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+    color: $color-white;
+    margin-bottom: 100px;
```
Comment (jules-js) [inline app/scss/modules/_team.scss]: again, remove this margin and add padding to the next section

File: app/scss/modules/_specialties.scss [hunk 14]
```diff
@@ -0,0 +1,33 @@
+.specialties{
```
Comment (jules-js) [inline app/scss/modules/_specialties.scss]: space before {

File: app/scss/modules/_photo.scss [hunk 15]
```diff
@@ -0,0 +1,7 @@
+.photo {
+    margin-bottom: 100px;
+    &__img {
+        max-width: 360px;
+
```
Comment (jules-js) [inline app/scss/modules/_photo.scss]: empty line

File: app/index.html [hunk 16]
```diff
@@ -5,12 +5,489 @@
...
+                            <button class="button button_yellow uppercase book__button" type="submit">BOOK NOW</button>
+                        </form>
+                    </div>
+                    <div class="book__img flex_basis-50">
+                        <img class="image" src="assets/images/book-img.png" alt="clock">
+                        <div class="square square_right"></div>
+                    </div>
+                </div>
+                <div class="schedule book__schedule text-center">
+                    Mon - Fri: 8PM - 10PM, Sat - Sun: 8PM - 3AM, Phone: +40 729 131 637/+40 726 458 782
```
Comment (jules-js) [inline app/index.html]: Phone - a href="tel:"

File: app/scss/modules/_contact.scss [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+        font-family: $o-font-bold,sans-serif;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 12px;
+        line-height: 12px;
+        align-items: center;
+        color: rgba(51, 51, 51, 0.5);
+        display: inline-block;
+        padding-top: 25px;
+        padding-left: 40px;
```
Comment (jules-js) [inline app/scss/modules/_contact.scss]: do spaces normal, please
we don't use padding for it

File: app/index.html [hunk 18]
```diff
@@ -5,12 +5,489 @@
...
+                        <input class="form__item form__item-contact" type="number" name="tel" placeholder="Phone"
+                            required>
+                    </div>
+
+                    <div class="flex_basis-100">
+                        <textarea class="form__item form__item-message" type="text" placeholder="Message"></textarea>
+                    </div>
+                    <div class="flex_basis-50">
+                        <a class="contact__nav contact__nav-gps" href="#">5th London Boulevard, U.K.</a>
+                        <a class="contact__nav contact__nav-tel" href="tel:+40729131637">+40 729 131 637/+40 726 458
```
Comment (jules-js) [inline app/index.html]: seriously?

File: app/index.html [hunk 19]
```diff
@@ -5,12 +5,489 @@
...
+                            782</a>
+                        <a class="contact__nav contact__nav-mail"
+                            href="mailto:office@mindblister.com">offmail@mindblister.com</a>
```
Comment (jules-js) [inline app/index.html]: where are icons before contact elements?

File: app/index.html [hunk 20]
```diff
@@ -5,12 +5,489 @@
...
+        </section>
+        <section class="map">
+            <iframe
+                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3482.219808201833!2d-0.14162588710522006!3d51.455853345374415!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4876044b741fd263%3A0xba9ce65afcade897!2sThe%20Abbeville!5e0!3m2!1suk!2sua!4v1624440923747!5m2!1suk!2sua"
+                width="100%" height="450" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="container">
+            <div class="footer__inner flex flex_center flex_align-center">
```
Comment (jules-js) [inline app/index.html]: footer__copyrigths

File: app/index.html [hunk 21]
```diff
@@ -5,12 +5,489 @@
...
+                </div>
+            </div>
+        </section>
+        <section id="photo" class="photo flex flex_center flex_nowrap">
+            <img class="photo__img" src="assets/images/photo-1.jpg" alt="hall">
+            <img class="photo__img" src="assets/images/photo-2.jpg" alt="fireplace">
+            <img class="photo__img" src="assets/images/photo-3.jpg" alt="hall">
+            <img class="photo__img" src="assets/images/photo-4.jpg" alt="hall">
+        </section>
+        <section id="contact" class="contact">
```
Comment (jules-js) [inline app/index.html:389]: Screenshot from 2021-07-04 15-38-58
Screenshot from 2021-07-04 15-38-52

fix according to the design

File: app/index.html [hunk 22]
```diff
@@ -5,12 +5,489 @@
...
+                <h2 class="title text-center contact__title">
+                    CONTACT
+                </h2>
+                <div class="text text__strong about__text text-center">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis at velit maximus, molestie
+                    est
+                    a, tempor magna.
+                </div>
+                <form class="form flex flex_wrap flex_center" action="#" method="get">
+                    <div class="flex_basis-33">
```
Comment (jules-js) [inline app/index.html]: Screenshot from 2021-07-04 15-40-10

add column with paddings, remove margin
input shouldn't respond for spaces between other elements

File: app/index.html [hunk 23]
```diff
@@ -5,12 +5,489 @@
...
+                <h2 class="title text-center menu__title">
+                    DELICIOUS MENU
+                </h2>
+                <div class="text text__strong about__text text-center">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis at velit maximus, molestie
+                    est
+                    a, tempor magna.
+                </div>
+                <nav>
+                    <ul class="menu__list flex flex_center text-center">
```
Comment (jules-js) [inline app/index.html]: .menu__tabs

File: app/index.html [hunk 24]
```diff
@@ -5,12 +5,489 @@
...
+                        <li class="menu__list-item"><a class="menu__list-link" href="#">SOUPE</a></li>
```
Comment (jules-js) [inline app/index.html]: .menu__tab-item

File: app/index.html [hunk 25]
```diff
@@ -5,12 +5,489 @@
...
+                        <li class="menu__list-item"><a class="menu__list-link" href="#">DESERT</a></li>
+                        <li class="menu__list-item"><a class="menu__list-link" href="#">WINE</a></li>
+                        <li class="menu__list-item"><a class="menu__list-link" href="#">BEER</a></li>
+                        <li class="menu__list-item"><a class="menu__list-link" href="#">DRINKS</a></li>
+                    </ul>
+                </nav>
+                <div class="menu__items">
+                    <div class="menu__item flex flex_wrap">
+                        <div class="menu__item-text">
+                            <p class="menu__item-name"> PIZZA QUATRO STAGIONI <span class="menu__item-dots">. . .
```
Comment (jules-js) [inline app/index.html]: are you sure this should be as paragraph?

File: app/scss/modules/_events.scss [hunk 26]
```diff
@@ -0,0 +1,11 @@
+.events{
+    background-image: url(../assets/images/events-bg.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
```
Comment (jules-js) [inline app/scss/modules/_events.scss]: why almost all sections have height 100vh? this is not according to the design

File: app/index.html [hunk 27]
```diff
@@ -5,12 +5,489 @@
...
+                    <div class="about__img flex_basis-50">
+                        <img class="image" src="assets/images/about-img.png" alt="fire">
+                        <div class="square square_right"></div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="team" class="team">
+            <div class="container">
+                <div class="sup-title uppercase text-center">
```
Comment (jules-js) [inline app/index.html:111]: subtitle

File: app/index.html [hunk 28]
```diff
@@ -5,12 +5,489 @@
...
+            </div>
+        </section>
+        <section id="book" class="book">
+            <div class="container">
+                <div class="flex flex_space-between text-center">
+                    <div class="flex_basis-50">
+                        <h2 class="title book__title uppercase">
+                            book a table
+                        </h2>
+                        <form class="form book__form flex flex_wrap flex_center" action="#" method="get">
```
Comment (jules-js) [inline app/index.html:143]: Screenshot from 2021-07-04 15-50-49

fix spaces between elements please

File: app/index.html [hunk 29]
```diff
@@ -5,12 +5,489 @@
...
+                    Mon - Fri: 8PM - 10PM, Sat - Sun: 8PM - 3AM, Phone: +40 729 131 637/+40 726 458 782
+                </div>
+            </div>
+        </section>
+        <section id="specialties" class="specialties">
+            <div class="container">
+                <div class="sup-title uppercase text-center">
+                    specialties
+                </div>
+                <!-- <div class="slider"> -->
```
Comment (jules-js) [inline app/index.html]: remove commented code

File: app/index.html [hunk 30]
```diff
@@ -5,12 +5,489 @@
...
+                <div class="flex flex_space-between text-center">
+                    <div class="book__img flex_basis-50">
+                        <img class="image" src="assets/images/specialties-img.png" alt="clock">
+                        <div class="square square_left"></div>
+                    </div>
+                    <div class="team__content flex_basis-50 text-center">
+                        <h2 class="title about__title">
+                            chocolate pancakes
+                        </h2>
+                        <div class="text text__strong about__text">
```
Comment (jules-js) [inline app/index.html]: text_strong

File: app/index.html [hunk 31]
```diff
@@ -5,12 +5,489 @@
...
+                            Integer ullamcorper neque eu purus euismod, ac faucibus mauris posuere. Morbi non
+                            ultrices
+                            ligula. Sed dictum, enim sed ullamcorper feugiat, dui odio vehicula eros, a sollicitudin
+                            lorem quam nec sem. Mauris tincidunt feugiat diam convallis pharetra. Nulla facilisis
+                            semper
+                            laoreet.
+                        </div>
+                    </div>
+                </div>
+                <!-- <div class="flex flex_space-between text-center">
```
Comment (jules-js) [inline app/index.html]: remove commented code

**Review by jules-js — CHANGES_REQUESTED** [review]

File: app/index.html (hunk 5 above)
Comment (Roman240594) [reply app/index.html]: sup-title - це як надзаголовок

**Pull request merged**